src/components/Buttons.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useMemo } from "react";
+import { ToggleButton, ToggleButtonGroup } from "@mui/material";
+import {
+  ViewStream,
+  Subject,
+  Edit,
+  Preview,
+  Undo,
+  Redo, Save
```

Comment:
The save button does not indicate when there are saves pending. Could be probably be tied to the length of the history?